networkx/algorithms/chordal/chordal_alg.py

```diff
@@ -242,6 +244,47 @@ def chordal_graph_treewidth(G):
         max_clique = max(max_clique,len(clique))
     return max_clique - 1
 
+@not_implemented_for('directed','multigraph')
```

Comment:
This needs to be two separate decorators, `@not_implemented_for('directed')` and on the next line `@not_implemented_for('multigraph')` to accomplish your intended goal.